### PR TITLE
[refactor](config) Delete the environment variable enable_vectorized_engine

### PR DESCRIPTION
**Pull request by Mryange**
# Proposed changes
Delete the environment variable enable_vectorized_engine

## Problem summary
remove variable enable_vectorized_engine in fe and regression-test.
Now using set enable_vectorized_engine=true in sql will result in an error('ERROR 1228 (HY000): errCode = 2, detailMessage = Unknown system variable 'enable_vectorized_engine'')

## Checklist(Required)

* [x] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zclllyybb** [comment]
please fix your title and describe your change

File: fe/fe-core/src/main/java/org/apache/doris/planner/StreamLoadPlanner.java [hunk 1]
```diff
@@ -273,7 +273,6 @@ public TExecPlanFragmentParams plan(TUniqueId loadId) throws UserException {
         queryOptions.setMemLimit(taskInfo.getMemLimit());
         // for stream load, we use exec_mem_limit to limit the memory usage of load channel.
         queryOptions.setLoadMemLimit(taskInfo.getMemLimit());
-        queryOptions.setEnableVectorizedEngine(Config.enable_vectorized_load);
```
Comment (HappenLee) [inline fe/fe-core/src/main/java/org/apache/doris/planner/StreamLoadPlanner.java:276]: do not change load

File: gensrc/thrift/PaloInternalService.thrift [hunk 2]
```diff
@@ -151,8 +151,6 @@ struct TQueryOptions {
   // if the right table is greater than this value in the hash join,  we will ignore IN filter
   34: optional i32 runtime_filter_max_in_num = 1024;
 
-  // whether enable vectorized engine 
-  41: optional bool enable_vectorized_engine = false
```
Comment (HappenLee) [inline gensrc/thrift/PaloInternalService.thrift:155]: change the variable to `true`

**Comment by zclllyybb** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 3]
```diff
@@ -1325,14 +1321,6 @@ public void setRuntimeFilterMaxInNum(int runtimeFilterMaxInNum) {
         this.runtimeFilterMaxInNum = runtimeFilterMaxInNum;
     }
 
-    public boolean enableVectorizedEngine() {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:1328]: Please keep this variable

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Review by HappenLee — DISMISSED** [review]
LGTM

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**